src/Subscriber/GitRepoSubscriber.php

```diff
@@ -291,7 +291,7 @@ private function initializeIssueTracker(ConsoleCommandEvent $event, $org, $repo,
 
         $issueTracker->authenticate();
 
-        /** @var \Gush\Adapter\BaseIssueTracker $issueTracker */
+        /* @var \Gush\Adapter\BaseIssueTracker $issueTracker */
```

Comment:
This phpdoc should be either deleted, or moved to the  actual variable assignment.